js/scripts.js

```diff
@@ -79,4 +78,385 @@ document.querySelector('.contact_mobile').addEventListener('click',() =>{
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    featuredImage: './images/Snapshoot-Portfolio-3.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.",
+    technologies: ['', 'html', 'css', 'javascript'],
+    linkToLiveVersion: '',
+    linkToSource: ''
+    },
+    {
+      title: 'Multi-Post Stories',
+      experiences: ['Canopy', 'Back End Dev', '2015'],
+      featuredImage: './images/Snapshoot-Portfolio-4.png',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required.",
+      technologies: ['', 'html', 'css', 'javascript'],
+      linkToLiveVersion: '',
+      linkToSource: ''
+    }
+]
+
+if((window.matchMedia('(max-width: 767.98px')).matches) {
+  console.log('logged');
+  const gridsContainer = document.querySelector('.grids-wrapper');
+  for(let i = 0; i < projects.length; i++) {
+    let gridDiv = document.createElement('div');
+    gridDiv.className = `grid grid${i + 1}`;
+    let li1 = '';
+    let li2 = '';
+    //const project = projects[i];
+    const experiences = projects[i].experiences;
+    const techs = projects[i].technologies;
+    for (let j = 0; j < experiences.length; j++) {
+      li1 += `<li class="experience-item">${experiences[j]}</li>`;
+    }
+
+    for (let k = 0; k < techs.length; k++) {
+      li2 += `<li class="tech">${techs[k]}</li>`;
+    }
+
+    gridDiv.innerHTML = `
+            <div class="grid-image image${i + 1}">
+              <img src="${projects[i].featuredImage}" alt="">
+            </div>
+
+            <h2 class="grid-title">${projects[i].title}</h2>
+            <ul class="experience-container">
+              ${li1}
+            </ul>
+            <p class ="project-description"> 
+              ${projects[i].description}
+            </p>
+            <div class = "technologies"
+              <ul class = "tech-container>
+                ${li2}
+              </ul>
+            </div>
+          
+              <div class="action-grid">
+                <button whichProjectBtn = ${i} class="action-grid-btn" type = "button">
+                  See Project
+                </button>
+              </div>
+          
+          </div>`;
+    gridsContainer.append(gridDiv);
+  }
+
+  // Popup window for mobile version starts here
+  const popupWindow = document.querySelector('.popup-card-wrapper');
+  function showPopupWindow() {
+    const btn = document.querySelectorAll('.action-grid-btn');
+    btn.forEach(function (e) {
+      e.addEventListener('click', function() {
+        const index = e.getAttribute('whichProjectBtn');
+        popupWindow.classList.toggle('hide');
+        const experiencePopup = projects[index].experiences;
+        const techpopup = projects[index].technologies;
+        let li1 = '';
+        for (let i = 0; i < experiencePopup.length; i++) {
+          li1 += `
+            <li class="popup-exp-name">${experiencePopup[i]}</li>
+          `;
```

Comment:
- [ ] Please kindly fix the pop-up window. I noticed it is not coming up at all when the `see project` button is clicked.

Reply:
Hi @Kamba56.
How are you? I used Firefox browser, VS code live server. I tested it for both the mobile version and desktop version. When I click the 'See project' button for each project and for both desktop version, it works. Kindly see it again please.